style.css

```diff
@@ -0,0 +1,309 @@
+@import 'https://fonts.googleapis.com/css?family=Lato:400,700';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(156, 88, 88);
```

Comment:
your website looks perfect but it could be nicer if you remove the body background color or use a background image or color that allows the page contents to be viewed clearly